2019/12/11 07:48:06 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: symfony.localhost, request: "GET /branch HTTP/1.1", upstream: "fastcgi://172.18.0.4:9001", host: "symfony.localhost:8080"
2019/12/11 07:48:18 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9001", host: "symfony.localhost:8080"
2019/12/11 07:56:06 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9001", host: "symfony.localhost:8080"
2019/12/11 07:56:06 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9001", host: "symfony.localhost:8080"
2019/12/11 07:58:04 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9001", host: "localhost:8080"
2019/12/11 07:58:07 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9001", host: "localhost:8080"
2019/12/11 07:58:08 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9001", host: "localhost:8080"
